pulsar-client-cpp/python/pulsar_test.py

```diff
@@ -1248,6 +1250,35 @@ def test_json_schema_encode(self):
         second_encode = schema.encode(record)
         self.assertEqual(first_encode, second_encode)
 
+    def test_logger_thread_leaks(self):
+        def _do_connect(close):
+            logger = logging.getLogger(str(threading.currentThread().ident))
```

Comment:
```suggestion
            logger = logging.getLogger(str(threading.current_thread().ident))
```

`currentThread` is deprecated, see https://docs.python.org/3/library/threading.html#threading.current_thread